### PR TITLE
Extract PouchDB storage into @deephaven/pouch-storage package

### PR DESCRIPTION
**Pull request by vbabich**

### CODE REVIEW

**Comment by mattrunyon** [comment]
Should probably add this to the root tsconfig as a project reference since they're all listed there. Although it probably won't affect anything since code-studio consumes it and it will get built by tsc just building code-studio

I think they were all listed there because it's one of the examples from typescript for projects in a monorepo

File: packages/pouch-storage/src/PouchCommandHistoryStorage.ts [hunk 1]
```diff
@@ -1,14 +1,14 @@
 /* eslint-disable class-methods-use-this */
 import Log from '@deephaven/log';
+import {
```
Comment (mofojed) [inline packages/pouch-storage/src/PouchCommandHistoryStorage.ts:3]: I'm a little confused about these re-ordered imports - why are they re-ordered?

**Review by mofojed — CHANGES_REQUESTED** [review]

**Comment by mofojed** [comment]
Also need to do an `npm install` to update the package-lock

File: packages/pouch-storage/src/PouchCommandHistoryStorage.ts (hunk 1 above)
Comment (vbabich) [reply packages/pouch-storage/src/PouchCommandHistoryStorage.ts:3]: Must be a result of cherry-picking from `0.13` to `main`.  Cleaned up.

**Review by mofojed — APPROVED** [review]

**Pull request merged**